cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -28,14 +29,42 @@
   @SerializedName("scheduler.queue.name")
   private final String schedulerQueueName;
 
-  public NamespaceConfig(String schedulerQueueName) {
+  @SerializedName("hdfs.directory")
+  private final String hdfsDirectory;
```

Comment:
can we just call this filesystemDirectory/fsDirectory, since we support other filesystems as well?